### PR TITLE
README assignment

### PR DESCRIPTION
**Pull request by BarBenNaim**
In this document, I explained about our project.
If you have anything to add you are welcome.
fixd #1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,14 @@
...
+allow people the ability to search for jobs over the internet directly from the hiring company
+
+####Description:
+a website that people will be able to sign in, set search criteria,
+and the system will find the matches from all the participating company sites.
+
+####How it works:
+1. The main page website that users will be able to sign in, and fill in their preferences.
+2. Database that contains all the users and their preferences + search history.
+3. n algorithm that searches over the internet (API) for a company career that establishes their jobs.
```
Comment (simzacks) [inline README.md:14]: 3. A backend that searches participating companies career websites for search matches

File: README.md [hunk 2]
```diff
@@ -1 +1,14 @@
-# ultimatejob
\ No newline at end of file
+#Job search engine:
+##UltimateJob.com
+
+####The goal is:
+allow people the ability to search for jobs over the internet directly from the hiring company
```
Comment (simzacks) [inline README.md:5]: hiring companies

**Review by simzacks — COMMENTED** [review]
Good start. I made a few minor suggestions

**Comment by Yarboa** [comment]
Please link the PR to an issue.
Please, next Issue elaborate more bit more

**Comment by BarBenNaim** [comment]
I think we are not allowed to edit the reviewers and linked issues

**Comment by simzacks** [comment]
@BarBenNaim - to link a PR to an issue - https://docs.github.com/en/free-pro-team@latest/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue

**Pull request closed without merging**